eya_def_tools/eya_def_tools/data_models/general.py

```diff
@@ -67,16 +67,56 @@ class Organisation(EyaDefBaseModel):
     )
 
 
-class AssessmentBasis(StrEnum):
-    """Basis on which an EYA or WRA component has been assessed."""
+AssessmentComponentBasis = Literal[
```

Comment:
For simple sets of string choices, it is not necessary to define a string enum class, but a string literal type is sufficient.